src/main/java/org/broadinstitute/hellbender/tools/exome/conversion/allelicbalancecaller/AllelicSplitCall.java

```diff
@@ -1,29 +1,29 @@
-package org.broadinstitute.hellbender.tools.exome.cnlohcaller;
+package org.broadinstitute.hellbender.tools.exome.conversion.allelicbalancecaller;
 
 import htsjdk.samtools.util.Locatable;
 import org.broadinstitute.hellbender.tools.exome.ACNVModeledSegment;
 
 import java.io.Serializable;
 
-public class CNLOHCall implements Serializable, Locatable{
+public class AllelicSplitCall implements Serializable, Locatable {
```

Comment:
Again, still not sure about the name AllelicSplitCall.  To me, this sounds like it should just be the opposite of AllelicBalanceCall.  However, it actually represents AllelicBalanceCall, CNLoHCall, and all of the other quantities.  I'll leave it up to you as to whether or not you want to change it.